### PR TITLE
migrate AdjustStackSpec from Spek to Kotest

### PR DESCRIPTION
**Pull request by maxslimb**
Migrated the AdjustStackSpec file from Spek to Kotest using Describe Spec.

It Resolves: #1201 
______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/main/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: atrium-core/src/jvmTest/kotlin/ch/tutteli/atrium/reporting/erroradjusters/AdjustStackSpec.kt [hunk 1]
```diff
@@ -13,12 +13,13 @@ import ch.tutteli.atrium.logic._logic
 import ch.tutteli.atrium.logic.creating.RootExpectBuilder
 import ch.tutteli.atrium.logic.utils.expectLambda
 import ch.tutteli.atrium.reporting.AtriumErrorAdjuster
+import io.kotest.core.spec.style.DescribeSpec
 import org.spekframework.spek2.Spek
 import org.spekframework.spek2.style.specification.describe
```
Comment (robstoll) [inline atrium-core/src/jvmTest/kotlin/ch/tutteli/atrium/reporting/erroradjusters/AdjustStackSpec.kt:18]: Please remove the spek imports

**Review by robstoll — CHANGES_REQUESTED** [review]

**Comment by robstoll** [comment]
@maxslimb migrating is quite easy isn't it 🙂
Please also cleanup and remove the spek specifics

File: atrium-core/src/jvmTest/kotlin/ch/tutteli/atrium/reporting/erroradjusters/AdjustStackSpec.kt (hunk 1 above)
Comment (maxslimb) [reply atrium-core/src/jvmTest/kotlin/ch/tutteli/atrium/reporting/erroradjusters/AdjustStackSpec.kt]: I have done this please review it

**Review by robstoll — APPROVED** [review]

**Comment by robstoll** [comment]
Take a look at https://github.com/robstoll/atrium/pull/1233/files
In case you want to see what would have been necessary

**Comment by Dark-Knight11** [comment]
Sure

**Pull request closed without merging**